trees/src/iavl/tree.rs

```diff
@@ -14,7 +14,7 @@ use crate::{error::Error, merkle::EMPTY_HASH};
 use super::node_db::NodeDB;
 
 #[derive(Debug, Clone, PartialEq, Default)]
-pub(crate) struct InnerNode {
+pub struct InnerNode {
```

Comment:
Does this need to be public outside the crate?

Reply:
Returned to private